front/app/components/admin/ActionForm/Fields/CustomField/index.tsx

```diff
@@ -31,6 +33,10 @@ const CustomField = ({ field, phaseId, action }: Props) => {
   const localize = useLocalize();
   const { formatMessage } = useIntl();
   const [isModalOpen, setIsModalOpen] = useState(false);
+  const isPermissionsCustomFieldsAllowed = useFeatureFlag({
```

Comment:
To stop it breaking for customers who have used custom fields where they should not have then maybe we could also check `permission. global_custom_fields` if this is true then no custom fields have been set for the permission so it is safe to disable everything?